cmake/Modules/FindWIL.cmake

```diff
@@ -0,0 +1,97 @@
+#[=======================================================================[.rst
+FindWIL
+-------
+
+FindModule for WIL and associated headers
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+.. versionadded:: 1.0.240122.1
```

Comment:
Apparently changed already, but correct: This currently reflects the version of our CMake build system it was added in, though in the future it might make sense to use the OBS version. But `3.0` would be correct right now.